### DISCUSSION TITLE
01improvement/00.index

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#770] [Announcements]
# 01improvement/00.index

 帮助文档 此文件夹，用于快速开发，便于自己对内容的查找。 文件说明 仪式感 上工仪式 打开电脑 整理工作台 泡茶 今年一天的工作日志书写。 对今天的工作安排。 浏览一下相关的网站，9 点半开始工作。 下班仪式 5 点开始整理今天的工作情况。 代码提交。 工作日志的完善， 今天完成的情况， 明天的安排， 本周的进度。 周五的时候看一下这个月的进度。 整...

http://localhost:1001/01improvement/00.index.html

**Comment by runkingzhang** [comment]
111
